tests/integration/loader/test_ext_grains.py

```diff
@@ -30,6 +31,18 @@ class LoaderGrainsTest(integration.ModuleCase):
     #    self.opts['grains'] = grains(self.opts)
 
     def test_grains_overwrite(self):
+        # To avoid a race condition on Windows, we need to make sure the
```

Comment:
This needs more investigation IMHO. What's supposed to be putting this file in the cache? Why is it taking so long? I very much dislike just putting in timeouts when it's not clear what the problem is.